.gitignore

```diff
@@ -4,6 +4,7 @@
 .env
 .DS_Store
 venv/
+.venv/
```

Comment:
The `.venv/` entry is correctly included to ignore virtual environment directories. However, ensure that this entry is necessary for your project setup. If you are using a different name for your virtual environment, adjust this line accordingly.